docs/docusaurus/versioned_docs/version-1.7.0/howtos/inbound_roaming.md

```diff
@@ -50,6 +50,47 @@ subscribers may also work but are not tested yet.
 Configurations with SubscriberDB, local HSS and remote HSS all at the same
 time is not supported yet.
 
+Next diagram shows you a logic diagram of Neutral Host architecture we will
+use on Inbound Roaming to be able to route the request properly. Note that
+in this diagram we add the Optional Local Feg. In case you are using
+subscriberDB for your local subscribers, you will not need that Feg Gateway,
+but you will still need the Neutral Host Network
+
+```mermaid
```

Comment:
Big thanks for the use of a mermaid diagram! :rocket: :heart: 
This is a lot better than importing bitmap-graphics!